CRITICAL - 2023-06-15 14:24:59 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-06-15 14:26:05 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
ERROR - 2023-06-15 12:29:34 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:29:34 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:30:12 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 12:30:12 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:31:32 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:31:32 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:31:49 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:31:49 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:32:22 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 12:32:22 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:34:06 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:34:06 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:35:24 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:35:24 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:35:42 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:35:42 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:47:31 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 12:47:31 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:48:01 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 12:48:01 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 12:59:26 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 12:59:26 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:04:06 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 13:04:06 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:04:34 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 13:04:34 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:31:31 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 13:31:31 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:45:42 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 13:45:42 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:45:58 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 13:45:58 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:46:36 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 13:46:36 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:54:04 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 13:54:04 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:55:43 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 13:55:43 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 13:56:17 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\Users\pasantedesarrollo\pasantias\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Register->do_register()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}
CRITICAL - 2023-06-15 13:56:17 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `password`) VALUES (:email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Register.php(26): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Register->do_register()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 14:00:28 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 14:00:28 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
ERROR - 2023-06-15 14:01:45 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectos_actas', 3306, '', 0)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'proyectos_actas' in C:\Users\pasantedesarrollo\pasantias\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\pasantedesarrollo\pasantias\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\pasantedesarrollo\pasantias\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\pasantedesarrollo\pasantias\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\Users\pasantedesarrollo\pasantias\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#6 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(934): App\Controllers\Login->do_login()
#7 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\Users\pasantedesarrollo\pasantias\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\pasantedesarrollo\pasantias\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\pasantedesarrollo\pasantias\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasant...')
#11 {main}
CRITICAL - 2023-06-15 14:01:45 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'proyectos_actas'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->do_login()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\pasantedesarrollo\\pasantias\\public\\index.php')
